app/components/viral/avatar_component.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Viral
+  # Avatar component to display on items first letter
+  class AvatarComponent < Viral::Component
+    attr_reader :initials, :size, :font_styles
+
+    SIZE_DEFAULT = :medium
+    SIZE_MAPPINGS = {
+      small: 'w-8 h-8',
+      medium: 'w-12 h-12',
+      large: 'w-16 h-16'
+    }.freeze
+
+    def initialize(name: nil, colour_string: nil, size: SIZE_DEFAULT, **system_arguments)
```

Comment:
**suggestion (non-blocking):** What do you think about optionally accepting a URL and then instead of adding a span we add a link tag so that clicking on the avatar also brings you to the Project for example.

Reply:
In the original version, that is how I had it, don't know why I left it out when I updated the component.  I think that it is a great idea and will update it now.

Reply:
Updated in 16e6e58